go.mod

```diff
@@ -62,4 +62,4 @@ require (
 	modernc.org/zappy v1.0.0 // indirect
 )
 
-go 1.13
+go 1.14
```

Comment:
revert this change since we still support go 1.13
https://golang.org/ref/mod#tmp_12

Reply:
Reverted